### DISCUSSION TITLE
node-chatgpt-api重复自我介绍已修复

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#40] [交流]
https://github.com/waylaidwanderer/node-chatgpt-api/pull/350
比我的客户端还少了多回复卡片的支持，防撤回也做得不好，但又不是不能用

**Comment by KaneBowman** [comment]
为什么更新node-chatgpt-api，按照原来的配置方法没法正常使用？

**Reply by sakyarz** [reply]
报错信息呢？

**Reply by KaneBowman** [reply]
报错：HttpsProxyAgent is not a constructor

**Reply by ghost** [reply]
pnpm install https-proxy-agent@~5.1.0~5.0.1

**Reply by KaneBowman** [reply]
pnpm install https-proxy-agent@5.1.0 这个怎么用？

**Reply by ghost** [reply]
在node-chatgpt-api目录下运行命令

**Reply by KaneBowman** [reply]
报错pnpm : 无法将“pnpm”项识别为 cmdlet、函数、脚本文件或可运行程序的名称。请检查名称的拼写，如果包括路径，请确保路径正确
，然后再试一次。

**Reply by ghost** [reply]
没用pnpm就用npm

**Reply by KaneBowman** [reply]
5.1.0报错

**Comment by KaneBowman** [comment]


**Reply by ghost** [reply]
反了，6.0.0报错

**Reply by KaneBowman** [reply]
不对，npm install https-proxy-agent@5.1.0运行后会提示npm ERR! code ETARGET
npm ERR! notarget No matching version found for https-proxy-agent@5.1.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

**Reply by ghost** [reply]
不好意思，看错了，是5.0.1

**Reply by KaneBowman** [reply]
谢谢你好心人，感激不尽，问题已经解决，但是有个问题新版不是已经更新6.0.0为什么安装时会报错？

**Reply by ghost** [reply]
新版做了不兼容的改动

**Reply by KaneBowman** [reply]
理解了，谢谢

**Comment by WarnError** [comment]
很奇怪，有些情况下会触发
[Error: The moderation filter triggered. Try again with different wording.]
然而我并没有输入敏感词
比如:"我不支持这项政策，因为它不能够保护用户的隐私。"
原版本最新文件不会这样
